src/Application/Filters/Actions/ActionWatershedFilter.cc

```diff
@@ -359,8 +347,8 @@ class WatershedFilterAlgo : public ITKFilter
 	virtual std::string get_layer_prefix() const
```

Comment:
The filter parameters are captured by the provenance system, so it's not necessary to have them be part of the layer name. The layer prefix should match format used by the other filters.